apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template-inserter.php

```diff
@@ -78,8 +78,17 @@ public function __construct( $theme_slug ) {
 
 	/**
 	 * Retrieves template parts content from WP.com API.
+	 *
+	 * @param boolean $should_fetch_from_api True if the template part data should be fetched from the API.
 	 */
-	public function fetch_template_parts() {
+	public function fetch_template_parts( $should_fetch_from_api = true ) {
```

Comment:
Optional nitpick: I think this could be shortened to `fetch_from_api` without losing too much info. Another way to improve readability could be using a constant on string here, e.g.:

`fetch_template_parts( BYPASS_API )` or `fetch_template_parts( 'bypass_api' )`

in which case the parameter could be named `fetch_strategy`. It's hard to tell at a glance what `fetch_template_parts( true )` is doing without looking at the code, or getting the PHP docs from the IDE.

Reply:
I added this to the class constructor :)